File: Java/Advanced Conditional Statements/Lab/11. Fruit Shop/FruitShop.java
import java.util.Scanner;

public class FruitShop {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        String product = input.nextLine(), day = input.nextLine();
        double amount = Double.parseDouble(input.nextLine());
        double total = 0;
        boolean valid = true;

        switch(day){
            case "Monday":
            case "Tuesday":
            case "Wednesday":
            case "Thursday":
            case "Friday":
                if(product.equals("banana")){
                    total = amount * 2.50;
                }else if(product.equals("apple")){
                    total = amount * 1.20;
                }else if (product.equals("orange")){
                    total = amount * 0.85;
                }else if(product.equals("grapefruit")){
                    total = amount * 1.45;
                }else if(product.equals("kiwi")){
                    total = amount * 2.70;
                }else if(product.equals("pineapple")){
                    total = amount * 5.50;
                }else if(product.equals("grapes")){
                    total = amount * 3.85;
                }else {
                    valid = false;
                }
                break;
            case "Saturday":
            case "Sunday":
                if(product.equals("banana")){
                    total = amount * 2.70;
                }else if(product.equals("apple")){
                    total = amount * 1.25;
                }else if (product.equals("orange")){
                    total = amount * 0.90;
                }else if(product.equals("grapefruit")){
                    total = amount * 1.60;
                }else if(product.equals("kiwi")){
                    total = amount * 3.00;
                }else if(product.equals("pineapple")){
                    total = amount * 5.60;
                }else if(product.equals("grapes")){
                    total = amount * 4.20;
                }else {
                    valid = false;
                }
                break;

            default:
                valid = false;
        }

        if(valid){
            System.out.printf("%.2f", total);
        }else {
            System.out.println("error");
        }

    }
}

File: Python/Basics/Conditional Statements Advanced/Lab/FruitShop.py
prices = [[2.50,1.20,0.85,1.45,2.70,5.50,3.85], [2.70,1.25,0.90,1.60,3.00,5.60,4.20]]

cost = 0

product = input()
day = input()
qty = float(input())
error = False

if day in "Monday Tuesday Wednesday Thursday Friday":
    if product == "banana":
        cost = qty * prices[0][0]
    elif product == "apple":
        cost = qty * prices[0][1]
    elif product == "orange":
        cost = qty * prices[0][2]
    elif product == "grapefruit":
        cost = qty * prices[0][3]
    elif product == "kiwi":
        cost = qty * prices[0][4]
    elif product == "pineapple":
        cost = qty * prices[0][5]
    elif product == "grapes":
        cost = qty * prices[0][6]
    else:
        print("error")
        error = True

elif day in "Saturday Sunday":
    if product == "banana":
        cost = qty * prices[1][0]
    elif product == "apple":
        cost = qty * prices[1][1]
    elif product == "orange":
        cost = qty * prices[1][2]
    elif product == "grapefruit":
        cost = qty * prices[1][3]
    elif product == "kiwi":
        cost = qty * prices[1][4]
    elif product == "pineapple":
        cost = qty * prices[1][5]
    elif product == "grapes":
        cost = qty * prices[1][6]

else:
    print("error")
    error = True

if not error:
    print(f"{cost:.2f}")